--- a/Makefile	2011-03-14 00:39:38.337013003 -0500
+++ b/Makefile	2011-03-14 00:39:52.603013003 -0500
@@ -1,5 +1,6 @@
 
 CC ?= gcc
+AR ?= ar
 CFLAGS ?= -g -O2
 
 prefix:=/usr
@@ -23,7 +24,7 @@
 	$(CC) -o $@ crypt.o -L. -lcrypt
 
 $(STATICLIB): encrypt.o
-	ar rv $@ encrypt.o
+	$(AR) rv $@ encrypt.o
 
 $(SHAREDLIB): encrypt.o
 	$(CC) -shared -Wl,--gc-sections -Wl,--out-implib=$(IMPORTLIB) crypt.def \
